app-modules/service-management/src/Http/Middleware/EnsureServiceManagementFeatureIsActive.php

```diff
@@ -38,14 +38,15 @@
 
 use Closure;
 use Illuminate\Http\Request;
+use Laravel\Pennant\Feature;
 use App\Settings\LicenseSettings;
 use Symfony\Component\HttpFoundation\Response;
 
 class EnsureServiceManagementFeatureIsActive
 {
     public function handle(Request $request, Closure $next): Response
     {
-        if (! app(LicenseSettings::class)->data->addons->serviceManagement) {
+        if (! Feature::active('service-request-feedback') && ! app(LicenseSettings::class)->data->addons->serviceManagement) {
```

Comment:
I think this check should be removed from here and instead put inside `ServiceRequestTypeFeedbackIsOn` and `FeedbackManagementIsOn`. Because I think if it is here then this will block the Service Request forms from loading till the feature flag is activated, and I don't think the Service Request forms rely on this data changes, do they?